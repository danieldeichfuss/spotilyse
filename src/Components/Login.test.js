import React from "react";
import ReactDOM from "react-dom";
import Login from "./Login";
import renderer from "react-test-renderer";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Login />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("renders page correctly", () => {
  const component = renderer.create(<Login />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
