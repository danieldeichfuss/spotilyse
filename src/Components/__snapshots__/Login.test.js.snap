// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component correctly 1`] = `
<div
  className="Login"
>
  <p
    className="Login__intro"
  >
    Take a deep dive into your soul ...
  </p>
  <Eyes />
  <p
    className="Login__intro"
  >
    ... I mean musical Soul
  </p>
  <Button
    text="Login with Spotify"
  />
</div>
`;
